.github/workflows/push-job-sched-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Upload Job-Scheduler Jar to Maven
+
+on:
+  push:
+    tags: 
+      - v*
+jobs:
+  upload-job-scheduler-jar:
+    runs-on: [ubuntu-16.04]
```

Comment:
I know job scheduler ran into issues with ubuntu-latest before, is that why you specifically chose 16.04?

Reply:
Yes, 16.04 LTS and stable